airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -532,6 +532,7 @@ def __init__(
         labels: dict | None = None,
         deferrable: bool = conf.getboolean("operators", "default_deferrable", fallback=False),
         poll_interval: float = 4.0,
+        bq_project_id: str | None = None,
```

Comment:
IMHO we should use `project_id` as we do in the other operators, wdyt?

Reply:
Are you suggesting to change the name of this variable from `bq_project_id` to `project_id` ? Sure, I will create a next rev for this

Reply:
fixed